tests/integration/ceph_csi_test.go

```diff
@@ -174,10 +224,9 @@ spec:
 	if !isPodRunning {
 		k8sh.PrintPodDescribe(namespace, csiTestRBDPodName)
 		k8sh.PrintPodStatus(namespace)
-	} else {
-		// cleanup the pod and pv
-		err = k8sh.ResourceOperation("delete", pod)
 	}
+	// cleanup the pod and pvc
+	err = k8sh.ResourceOperation("delete", pod)
 
 	require.True(s.T(), isPodRunning, "csi rbd test pod fails to run")
```

Comment:
Let's change this to `assert.True()`? Then the test won't be aborted when it fails. This is preventing the new test cleanup code from being executed. For example, see the test log [here](https://jenkins.rook.io/blue/rest/organizations/jenkins/pipelines/rook/pipelines/rook/branches/PR-3520/runs/3/nodes/53/steps/114/log/?start=0) and search for `TestBlockCSI_SmokeTest`. This is the log from the [1.13 tests](https://jenkins.rook.io/blue/organizations/jenkins/rook%2Frook/detail/PR-3520/3/pipeline/53).